### DISCUSSION TITLE
Cotes coupe de France

### DISCUSSION THREAD

**Discussion by kreon76** [pretrehr/Sports-betting#28] [General]
Pour info les cotes de la coupe de France correspondent à l'Italie : 

![image](https://user-images.githubusercontent.com/78261069/109952901-ae5fd780-7cdf-11eb-9061-a05e308297dc.png)

![image](https://user-images.githubusercontent.com/78261069/109952939-b7e93f80-7cdf-11eb-9fcf-ab385c62d1d8.png)

**Comment by pretrehr** [comment]
C'est corrigé, merci !
